README.md

```diff
@@ -1 +1,145 @@
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Follow a correct Gitflow**
+- **Diagram a data model clearly and explain what it's doing**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+✅ Github account <br>
+✅ Visual Studio Code installed <br>
+✅ Node.js installed <br>
+✅ Git Bash installed (optional) <br>
+✅ PostgreSQL installed  
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd database-clinic
+  git clone https://github.com/LarryIVC/database_clinic.git
+```
+
+### Install
+
+N/A.
+
+### Usage
+
+Run the SQL Shell (psql) or the pgAdmin 4.
+
+### Run tests
+
+N/A.
+
+### Deployment
+
+N/A.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+**Larry Villegas**
+- GitHub: [@LarryIVC](https://github.com/LarryIVC)
+- Twitter: [@LarryIVC](https://twitter.com/LarryVillegas)
+- LinkedIn: [@LarryIVC](https://www.linkedin.com/in/larryvillegascostas/)
+
+- **Carlos Igreda**
+- GitHub: [@CarlosIgreda](https://github.com/CarlosIgreda)
+- Twitter: [@carlosigreda](https://twitter.com/carlosigreda)
+- LinkedIn: [@carlosigreda](https://www.linkedin.com/in/carlosigreda/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Add more tables and data**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+👤 Collaboration:
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/LarryIVC/database_clinic/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can follow me on Github.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank all Microverse staff and my coding partners as well.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ❓ FAQ <a name="faq"></a>
+
+- **What is a query in SQL?**
+
+  - SQL queries are made up of commands that allow you to manipulate data within a database. These commands follow a specific syntax (a set of rules) so that they're interpreted correctly by the database management system (DBMS).
+
+- **What is meant by Gitflow?**
+
+  - Gitflow is an alternative Git branching model that involves the use of feature branches and multiple primary branches.
+  
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
- [ Optional ] I think it would be better if you fix this link on your readme file. Currently, the link leads to a `404` page not found error. I would like to recommend that you change the link to the one below because the name of your License file is `LICENSE` and not `LICENSE.md`

```sh
This project is [MIT](./LICENSE) licensed.
```

## Error 
![Screenshot from 2023-06-29 01-56-10](https://github.com/LarryIVC/database_clinic/assets/16392046/45f73b85-a3fb-45ba-b139-f8ea19c73788)

Reply:
Thanks, @Felix45, we corrected this issue.